cmd/root.go

```diff
@@ -52,7 +55,7 @@ func init() {
 func NewRootCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:           "pingctl",
-		Version:       "v2.0.0-alpha.3",
+		Version:       "v2.0.0-alpha.4",
```

Comment:
In the providers we have a way where these version numbers get inserted at build time, so you wouldn't have to update a hard-coded string in here - might be useful. https://github.com/pingidentity/terraform-provider-pingfederate/blob/81430ae0e8962ca3edd3720f9b0cf1002e78a4c8/main.go#L26

Reply:
That is very helpful. I'm going to break that out into a future PR.